app/questionnaire/answer_store_updater.py

```diff
@@ -1,3 +1,4 @@
+from typing import List
```

Comment:
Now that this is updating the list store, in addition to completed blocks, we should rename it `questionnaire_store_updater`.